crypto/s2n_hash.h

```diff
@@ -87,6 +87,7 @@ struct s2n_hash {
     int (*free) (struct s2n_hash_state *state);
 };
 
+const EVP_MD* s2n_hash_alg_to_evp_md(s2n_hash_algorithm alg);
```

Comment:
How about adding a proof harness for this function as well? It should be pretty similar too `s2n_hash_is_available`.

Reply:
I'm not going to block this PR on that bc it's not a particularly interesting or dangerous method, but I can create an issue and do a follow up.

Reply:
https://github.com/aws/s2n-tls/issues/3127